react/features/prejoin/functions.js

```diff
@@ -158,7 +158,7 @@ export function isJoinByPhoneDialogVisible(state: Object): boolean {
  */
 export function isPrejoinPageEnabled(state: Object): boolean {
     return navigator.product !== 'ReactNative'
-        && state['features/base/config'].prejoinPageEnabled
+        && state['features/base/config'].prejoinConfig.enabled
```

Comment:
```suggestion
        && state['features/base/config'].prejoinConfig?.enabled
```

Reply:
`prejoinConfig` is always set, at least as an empty object , in the config reducer

Reply:
Ah true.

Reply:
Wait. Yes it is... for now. 6 months from now, when we remove the backwards compat code it won't. Please take this into account.

Reply:
good point, I'll add the check.